calaccess_raw/management/commands/loadcalaccessrawfile.py

```diff
@@ -3,60 +3,55 @@
 from django.apps import apps
 from csvkit import CSVKitReader
 from django.conf import settings
-from optparse import make_option
 from postgres_copy import CopyMapping
 from django.db import connections, router
 from calaccess_raw.management.commands import CalAccessCommand
-from django.core.management.base import LabelCommand, CommandError
-
-
-custom_options = (
-    make_option(
-        "-a",
-        "--app-name",
-        action="store",
-        type="string",
-        dest="app_name",
-        default="calaccess_raw",
-        help="Name of Django app where model will be imported from"
-    ),
-    make_option(
-        "-c",
-        "--csv",
-        action="store",
-        type="string",
-        dest="csv",
-        default=None,
-        help="Path to comma-delimited file to be loaded. Defaults to one associated with model."
-    ),
-    make_option(
-        "-d",
-        "--database",
-        action="store",
-        type="string",
-        dest="database",
-        default=None,
-        help="Alias of database where data will be inserted. Defaults to the 'default' database."
-    ),
-)
-
-
-class Command(CalAccessCommand, LabelCommand):
+from django.core.management.base import CommandError
+
+
+class Command(CalAccessCommand):
     help = 'Load clean CAL-ACCESS file into its corresponding database model'
-    option_list = CalAccessCommand.option_list + custom_options
-    args = '<model name>'
+
+    def add_arguments(self, parser):
```

Comment:
Perhaps a good idea to call `super` here, just in case `CalAccessCommand` has options to add as well?